analyses/rnaseq-batch-correct/README.md

```diff
@@ -0,0 +1,160 @@
+## Batch Correction of RNA-seq expression matrices
+
+**Module authors:** Komal Rathi ([@komalsrathi](https://github.com/komalsrathi))
+
+### Description
+
+The goal of this analysis is to combine and batch-correct expression matrices from various RNA-seq datasets. 
+The module generates combined uncorrected and batch-corrected matrices as well as combined metadata files that has the sample identifier, batch information and other specified columns only.
+For QC, the module also generates UMAP/t-SNE clustering of most variable genes and housekeeping genes before and after correction as well as density plots with housekeeping genes.
```

Comment:
Can you add some additional information about this? How were those housekeeping genes chosen? Is there a publication we can add about them?

Reply:
@jharenza These are some known housekeeping genes that came up during a discussion and not from any specific literature: https://github.com/d3b-center/bixu-tracker/issues/675#issuecomment-642613755 
But I can certainly re-run with a set that can be linked to a publication.

Reply:
Thanks @aadamk! I will add those and update.